lib/stat_tracker.rb

```diff
@@ -689,3 +572,30 @@ def rival(team_name)
 		return fav_opponent_team.team_name
 		end
 	end
+
+
+	# def highest_scoring_home_team
```

Comment:
Does this commented out code need to be saved for later or can it be deleted?

Reply:
It's okay to be deleted. Thanks Thomas!